src/device/GPU_Device.hh

```diff
@@ -106,17 +106,6 @@ public:
    *  textureAlign bytes do not need an offset applied to texture fetches */
   int textureAlign(int devId = 0) const { return m_texturealign[devId]; }
 
```

Comment:
At the top of this file, replace this doxygen example reference with one of the new files.

```
 * \example device/test/gpu_hello_driver_api.cc
 * Test of GPU_Device for CUDA driver environment.
```

This should fix the autodoc build failure.

Reply:
I've removed this reference to a test file for the old driver API.